modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -23,12 +24,10 @@ Adding the {microshift-short} RPM package to an image builder blueprint enables
 . Use the following example to create your own blueprint file:
 +
 .Custom image builder blueprint example
-+
-[source,text]
-[subs="+quotes"]
+[source,text,subs="+quotes",subs="+attributes"]
 ----
-cat > __<microshift_blueprint.toml>__ <<EOF <1>
-name = "__<microshift_blueprint>__" <2>
+cat > _<microshift_blueprint.toml>_ <<EOF <1>
```

Comment:
It's really inconvenient to have pseudo names as part of the command. 
Here, we're creating a file, so we can just decide to name it microshift_blueprint.toml without any additional signs.